Packs/DelineaSS/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,8 @@
+#### Integrations
+##### Delinea
+- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
+- Rebranded from Thycotic Secret Server to Delinea Secret Server
+##Bug Fixes
+- Not able to create secret using delinea-secret-create command
+- Secret Checkout command throws “List Index out of range” exception.
+

```

Comment:
Please remove all Release notes as this is a new pack (while the original will be deprecated.)